code/modules/mob/living/carbon/human/species/apollo.dm

```diff
@@ -45,6 +45,8 @@
 	base_color = "#704300"
 	flesh_color = "#704300"
 	blood_color = "#FFFF99"
+	//Default styles for created mobs.
+	default_hair = "Antennae"
```

Comment:
Means that, for example, when creating a mob from the admin game panel, they will spawn with this hair-style.